packages/x-components/static-docs/build-search-ui/web-how-to-use-x-components-guide.md

```diff
@@ -1,41 +1,46 @@
 ---
-title: Use & configure Interface X Components in your project
+title: How to use and configure Interface X Components in your project
 tags:
   - configuration
   - x configuration
   - x component configuration
+  - use x components
+  - how to
 ---
 
-# Use and configure Interface X Components in your project
+# How to use and configure Interface X Components in your project
```

Comment:
You are calling this section `Use and configure` at README.md, but here you are calling it `How To use and configure`here

Reply:
They are different. The individual page/file is called _How to use and configure..._ as we agreed. In the readme we are referring to the section within the getting started page, which doesn't need to be called the same. In addition, in this section, you find a link to the how-to with the appropriate title as follows: `Unsure how to start? Check out [How to use and configure Interface X Components in your project](/develop-empathy-platform/build-search-ui/web-how-to-use-x-components-guide.md).`